ci/ci.sh

```diff
@@ -83,7 +83,19 @@ esac
     --host="$HOST" $EXTRAFLAGS
 
 # We have set "-j<n>" in MAKEFLAGS.
-make
+build_exit_code=0
+make || build_exit_code=$?
+if [ $build_exit_code -ne 0 ]; then
+    case $CC in
+        *snapshot*)
+            make 2>&1 >/dev/null | grep "internal compiler error:"
```

Comment:
Regarding `grep "internal compiler error:"`:

Could also additionally grep for "PLEASE submit a bug report". This is what clang will output when it crashes.

Reply:
Sure. [Done](https://github.com/bitcoin-core/secp256k1/pull/1437#issuecomment-1798325434).